app/controllers/users_controller.rb

```diff
@@ -0,0 +1,36 @@
+class UsersController < ApplicationController
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+
+    respond_to do |format|
+      if @user.save
+        format.html { redirect_to login_url, notice: "#{@user.name} successfully created" }
+      else
+        format.html { render :new, notice: "#{@user.name} not got created" }
+      end
+    end
+  end
+
+  def verify
+    user = User.find(params[:id])
+
+    respond_to do |format|
+      if user
+        format.html { redirect_to login_url, notice: "User successfully verified" }
+      else
+        format.html { redirect_to login_url, notice: "User could not got verified" }
+      end
+    end
```

Comment:
As discussed, please use a token.
Do not use notice for error messages.
Avoid redundant format.html.